lms/exceptions.py

```diff
@@ -13,7 +13,19 @@ class LTILaunchError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
     pass
 
 
+class LTIContentItemError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
+    """Base exception for problems handling LTI content item selection."""
+
+    pass
+
+
 class MissingLTILaunchParamError(LTILaunchError):  # pylint: disable=too-many-ancestors
```

Comment:
Do all of these Exception classes really raise pylint too-many-ancestry errors (they might! I just was wondering if you'd checked without the disable on these)?